packages/spark-core/settings/_settings.scss

```diff
@@ -466,7 +480,7 @@ $dropdown-padding: $space-inset-short-l !default;
 $dropdown-header-padding: $space-inset-short-l !default;
 $dropdown-footer-padding: $space-inset-short-l !default;
 $dropdown-border: 1px solid $gray !default;
-$dropdown-active-background-color: $gray-tint-75 !default;
+$dropdown-active-background-color: $gray !default;
```

Comment:
Make sure you update the dropdown page Sass Var Table to reflect the new default color.